.github/workflows/ray_julia_jll_CI.yml

```
@@ -101,6 +101,13 @@ jobs:
           cp -rfp . "$RAY_GEN_CACHE_DIR"
         working-directory: ${{ env.ray_dir }}
 
+      - name: "JLL setup"
+        shell: julia --color=yes --project {0}
+        run: |
+          overrides_toml = joinpath(first(Base.DEPOT_PATH), "artifacts", "Overrides.toml")
+          isfile(overrides_toml) && rm(overrides_toml)
+          using Pkg
+          Pkg.instantiate(verbose=true)
```

Currently is working in the `Ray.jl` repo root:
```suggestion
          Pkg.instantiate(verbose=true)
          include("deps/build_jll.jl")
        working-directory: ${{ env.ray_julia_jll_dir }}
```

---

ah that makes sense